warrior/Actions/CliActions/cli_actions.py

```diff
@@ -816,25 +817,21 @@ def send_testdata_command_kw(self, system_name, session_name=None, wdesc='', var
         Utils.testcase_Utils.pNote("Datafile: {0}".format(self.datafile))
         session_id = Utils.data_Utils.get_session_id(system_name, session_name)
         session_object = Utils.data_Utils.get_object_from_datarepository(session_id)
-        testdata, varconfigfile = Utils.data_Utils.get_td_vc(self.datafile, system_name, td_tag, vc_tag)
+        testdata, varconfigfile = Utils.data_Utils.get_td_vc(self.datafile, system_name,
+                                                             td_tag, vc_tag)
         #abspaths = Utils.data_Utils.get_filepath_from_system(self.datafile, system_name,
         #                                                     'testdata', 'variable_config')
         #testdata = abspaths[0]
         #varconfigfile = abspaths[1]
         # td_resp_dict = {}
-        status, resp_dict = cli_Utils.send_commands_from_testdata(testdata, session_object,
+        status, resp_dict, td_resp_dict = cli_Utils.send_commands_from_testdata(testdata, session_object,
```

Comment:
If you need only td_resp_dict, why getting resp_dict as well?? Why the need for extra return parameter ??